logbook-json/src/main/java/org/zalando/logbook/json/JsonPathBodyFilters.java

```diff
@@ -81,6 +83,7 @@ private static JsonPathBodyFilter filter(final Operation operation) {
         private static final ParseContext CONTEXT = JsonPath.using(
                 Configuration.builder()
                         .jsonProvider(new JacksonJsonNodeJsonProvider())
+                        .mappingProvider(new JacksonMappingProvider())
```

Comment:
What does a MappingProvider do exactly? I tried this but couldn't observe any difference and the tests still worked.

Reply:
I do not know. I know the default MappingProvider use json-smart, so override it to be save.